lib/jenkiexs/builds.ex

```diff
@@ -12,38 +11,61 @@ defmodule Jenkiexs.Builds do
   def details(%Build{job_name: job_name, number: build_number}),
     do: details(job_name, build_number)
 
-  @spec details(Job.t(), Build.t() | binary() | integer()) :: {:ok, Build.t()} | {:error, binary()}
+  @spec details(Job.t(), Build.t() | binary() | integer()) ::
+          {:ok, Build.t()} | {:error, binary()}
   def details(%Job{name: job_name} = _job, %Build{number: build_number}),
     do: details(job_name, build_number)
+
   def details(%Job{name: job_name} = _job, build_number),
     do: details(job_name, build_number)
 
-  @spec details(job_name(), build_number :: binary() | integer()) :: {:ok, Build.t()} | {:error, binary()}
+  @spec details(job_name(), build_number :: binary() | integer()) ::
+          {:ok, Build.t()} | {:error, binary()}
   def details(job_name, build_number) do
     case Client.get!("/job/#{job_name}/#{build_number}/api/json") do
       %{status_code: 200, body: body} ->
         build = new(job_name, body)
         {:ok, build}
+
       response ->
         {:error, inspect(response)}
     end
   end
 
-  @spec last(Job.t() | job_name())
-    :: {:ok, Build.t()} | {:error, reason :: binary()}
+  @doc """
+  Returns the last job builded.
+
+  ## Examples
+
+      iex> Jenkiexs.Build.last(%Job{name: "example"})
+      {:ok, %Build{job_name: "example", number: 42, building: true, ...}}
+
+      iex> Jenkiexs.Build.last("example")
+      {:ok, %Build{job_name: "example", number: 42, building: true, ...}}
+
+      iex> Jenkiexs.Build.last("another_example")
+      {:error}
+
+  """
```

Comment:
@volcov would you mind to add a [Doctest](https://hexdocs.pm/ex_unit/1.10.0/ExUnit.DocTest.html#content) for this?
Thank you!